backend/src/models/users.js

```diff
@@ -3,15 +3,30 @@ const dbConnection = new DatabaseConnection();
 const db = dbConnection.knex;
 
 const table = `${dbConnection.dataBaseSchemas().public}.users`;
+const rolesTable = `${dbConnection.dataBaseSchemas().public}.roles`;
 
 // Get all.
 const findAll = () => {
-  return db(table);
+  return db
+    .select("users.id", "users.name", "users.email", "roles.display_name")
+    .from(table)
+    .leftJoin(rolesTable, { "public.users.role_id": `${rolesTable}.id` });
```

Comment:
Is it better to use aliases for the table names (`users as u`, then reference columns as `u.id`, `u.name`, etc.)? I did this in the Contacts model so that we don't have to repeat/hardcode the table names.

Reply:
i think this is a good point, however i know there is  optimization for all these forms, which is going to come, probably in next sprint.